### ISSUE TITLE
converting drawing to text string?

### ISSUE THREAD

**Issue by saffiah** [mozman/ezdxf#70]
Hi, 
I'm having trouble writing the finished drawing object as a string. After I've finished adding entities to the drawing, I can save it as a text file with dwg.saveas('writePoints.dxf'). But, I want to save the contents of that file as text string, without having to write the file. I thought that Drawing.write(stream) should give me what I want, but I can't seem to get this function to work. I've tried various ways, but running dwg.write('utf-8') gives me this error:

Traceback (most recent call last):
  File "writeDXF_StrInput.py", line 44, in <module>
    dwg.write('utf-8')
  File "C:\Python27\ArcGIS10.6\lib\site-packages\ezdxf\drawing.py", line 407, in write
    self.sections.write(tagwriter)
  File "C:\Python27\ArcGIS10.6\lib\site-packages\ezdxf\sections\sections.py", line 81, in write
    section.write(tagwriter)
  File "C:\Python27\ArcGIS10.6\lib\site-packages\ezdxf\sections\header.py", line 170, in write
    tagwriter.write_str("  0\nSECTION\n  2\nHEADER\n")
  File "C:\Python27\ArcGIS10.6\lib\site-packages\ezdxf\lldxf\tagwriter.py", line 42, in write_str
    self._stream.write(s)
AttributeError: 'str' object has no attribute 'write'

If anyone can help me with this in any way, I'd be greatly appreciative.

**Comment by saffiah** [comment]
Hi, Just as a quick follow up: I ended up creating a work-around that gives me the same final result (the contents of the dxf file to be written as a string). In case anyone is interested, I saved the drawing as a dxf file, then just re-open it as a string. 

dwg.saveas('write_point_att1.dxf')
dwgT = io.open('write_point_att1.dxf',mode='rt')
outputText = dwgT.read() #the final string consisting of the contents of the dxf file

Still not sure why I couldn't get dwg.write() to work, but this works similarly.
